packages/xchain-thorchain/src/util.ts

```diff
@@ -205,56 +185,6 @@ export const getChainIds = async (client: ClientUrl): Promise<ChainIds> => {
   }))
 }
 
-// /**
-//  * Structure StdTx from MsgNativeTx.
-//  *
-//  * @param {MsgNativeTx} msgNativeTx Msg of type `MsgNativeTx`.
-//  * @param {string} nodeUrl Node url
-//  * @param {chainId} ChainId Chain id of the network
-//  *
-//  * @returns {Tx} The transaction details of the given transaction id.
-//  *
-//  * @throws {"Invalid client url"} Thrown if the client url is an invalid one.
-//  */
-// export const buildDepositTx = async ({
```

Comment:
Any reason to remove `buildDepositTx`?  It's still needed for creating txs to sign it with Ledger (see Ledger implementation in ASGDX [here](https://github.com/thorchain/asgardex-electron/blob/0da65705dddfc8b1faedf87745c272eca3dfb32b/src/main/api/ledger/thorchain/transaction.ts#L212))

Reply:
yup my b, re-added with new return value (called this out in breaking changes)